### ISSUE TITLE
Unable to use keyboard navigation for custom slash commands

### ISSUE THREAD

**Issue by sandip-mane** [neetozone/neeto-editor#206]
Refer: https://github.com/bigbinary/neeto-desk-web/issues/12522
